root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -119,9 +119,9 @@ const flags = require('../../flags');
         sortname: guess("Work", "guessSortName")
     };
 
-    // Series doesn't have it's own handler, and just uses the work handler
-    // because additional behavior isn't needed.
+    // Series and Event doesn't have their own handler, and they use the work handler because additional behavior isn't needed.
```

Comment:
So, you missed one singular here (`doesn't`). :)

And it should remain in two lines; the question was only where to break it. You could do it after “use the work handler” (as before) or after “use the”.